### PR TITLE
fix: abortBars added

### PR DESCRIPTION
**Pull request by syncrosonic**
unblocks process if start was not received

### CODE REVIEW

File: ymodem/ymodem.go [hunk 1]
```diff
@@ -125,6 +125,14 @@ func ModemSend(c io.ReadWriter, bs int, files []File) error {
 
 	var err error
 
+	abortBars := func(){
```
Comment (troian) [inline ymodem/ymodem.go]: space before `{`

File: ymodem/ymodem.go [hunk 2]
```diff
@@ -125,6 +125,14 @@ func ModemSend(c io.ReadWriter, bs int, files []File) error {
...
+		for _, f := range files {
```
Comment (troian) [inline ymodem/ymodem.go]: use instead
```go
for i := range files {
	if !f[i].bytesBar.Completed() {
		f[i].bytesBar.Abort(true)
	}
}
```

File: ymodem/ymodem.go [hunk 3]
```diff
@@ -156,6 +166,7 @@ func ModemSend(c io.ReadWriter, bs int, files []File) error {
 		)
 	}
 
+//	pBars.Add()
```
Comment (troian) [inline ymodem/ymodem.go]: Remove commented code

File: ymodem/ymodem.go [hunk 4]
```diff
@@ -133,6 +141,8 @@ func ModemSend(c io.ReadWriter, bs int, files []File) error {
 
 	pBars := mpb.New(mpb.WithWidth(64))
 	defer pBars.Wait()
+	//to unblock if files were not sent
+	defer abortBars()
```
Comment (troian) [inline ymodem/ymodem.go]: better to combine abort and wait into finalize func

```go
finalizeBars := func() {
	for i := range files {
		if !f[i].bytesBar.Completed() {
			f[i].bytesBar.Abort(true)
		}
	}
	pBars.Wait()
}
```

File: ymodem/ymodem.go (hunk 3 above)
Comment (syncrosonic) [reply ymodem/ymodem.go]: It is not mine. Anyway, removed

File: ymodem/ymodem.go (hunk 1 above)
Comment (syncrosonic) [reply ymodem/ymodem.go]: fixed

File: ymodem/ymodem.go (hunk 2 above)
Comment (syncrosonic) [reply ymodem/ymodem.go]: fixed

File: ymodem/ymodem.go (hunk 4 above)
Comment (syncrosonic) [reply ymodem/ymodem.go]: fixed with closure

**Pull request merged**